PL_2_komentarze.py

```diff
@@ -0,0 +1,2722 @@
+
+    kolumnatempminimalna = Entry(matka, width=30)
+    kolumnatempminimalna.grid(padx=2, pady=2, sticky=E, row=7, column=1)
+
+    Label(matka, text='Średnica wewnętrzna (mm):').grid(padx=2, pady=2, sticky=W, row=8, column=0)
+
+    kolumnasrednica = Entry(matka, width=30)
+    kolumnasrednica.grid(padx=2, pady=2, sticky=E, row=8, column=1)
+
+    Label(matka, text='Ciśnienie robocze (bar):').grid(padx=2, pady=2, sticky=W, row=9, column=0)
+
+    kolumnacisnienierobocze = Entry(matka, width=30)
+    kolumnacisnienierobocze.grid(padx=2, pady=2, sticky=E, row=9, column=1)
+
+    Label(matka, text='Ciśnienie rozerwania (bar):').grid(padx=2, pady=2, sticky=W, row=10, column=0)
+
+    kolumnacisnienierozerwania = Entry(matka, width=30)
+    kolumnacisnienierozerwania.grid(padx=2, pady=2, sticky=E, row=10, column=1)
+
+    Label(matka, text='Promień zagięcia (statyczny) (mm):').grid(padx=2, pady=2, sticky=W, row=11, column=0)
+
+    kolumnapromienstatyczny = Entry(matka, width=30)
+    kolumnapromienstatyczny.grid(padx=2, pady=2, sticky=E, row=11, column=1)
+
+    Label(matka, text='Promień zagięcia (dynamiczny) (mm):').grid(padx=2, pady=2, sticky=W, row=12, column=0)
+
+    kolumnapromiendynamiczny = Entry(matka, width=30)
+    kolumnapromiendynamiczny.grid(padx=2, pady=2, sticky=E, row=12, column=1)
+
+    Label(matka, text='Flaga tolerancji:').grid(padx=2, pady=2, sticky=W, row=13, column=0)
+    Button(matka, text='?', command=pomocflaga).grid(sticky=E, row=13, column=0)
+
+    kolumnaflagatolerancji = ttk.Combobox(matka, state='readonly', width=27)
+    kolumnaflagatolerancji['values'] = ['P1', 'P2', 'P3', 'S1', 'S2', 'S3', 'S4', 'S5', 'K', 'C']
+    kolumnaflagatolerancji.grid(padx=2, pady=2, sticky=E, row=13, column=1)
+    kolumnaflagatolerancji.current(0)
+
+    Button(matka, text='P', command=lambda:otworztoler('P'), width=3).grid(row=14, padx=20, pady=6, sticky=W)
+    Button(matka, text='S', command=lambda:otworztoler('S'), width=3).grid(row=14, padx=20, pady=6)
+    Button(matka, text='K', command=lambda:otworztoler('K'), width=3).grid(row=14, padx=20, pady=6, sticky=E)
+    Button(matka, text='C', command=lambda:otworztoler('C'), width=3).grid(row=14, padx=20, pady=6, column=1, sticky=W)
+
+    Button(matka, text='Dodaj wąż!', height=2, command=dodajwazdolisty).grid(padx=2, pady=10, sticky=E, row=15, column=0, columnspan=1)
+
+    matka.mainloop()
+
+def pomocflaga():
+    messagebox.showinfo('Pomoc nt. flag tolerancji', 'Flagi tolerancji to oznaczenie, dzięki któremu na świadectwie pojawia się informacja o tolerancji wykonania przewodu.\n\nKategorie:\nP - dla przewodów \"podstawowych\"\nS - dla przewodów stalowych\nK - dla przewodów kompozytowych\nC - dla przewodów Corroflon\n\nCyfry przy znacznikach P i S oznaczają kategorię ze względu na średnicę, tak jak podają instrukcje tolerancji dla tych rodzajów przewodów.\n\nW razie wątpliwości kontakt z Mariuszem Węcławiakiem z KJ.')
+
+def checkifitsce():
```

Comment:
Każda funkcja typu boolean `is` `has` itp powinna zwracać `true` lub `false` zamiast liczb